fluent/src/resolver.js

```diff
@@ -288,6 +353,11 @@ function Type(env, expr) {
 /**
  * Resolve a reference to an external argument.
  *
+ * @param   {Object}  env
+ *    Resolver environment object.
+ * @param   {Object}  name
+ *    Name of an argument to be returned.
+ * @returns {T}
```

Comment:
`{T}` indicates that any type can be returned (I took it from Rust and C++) since if I understand correctly, ExternalArgument may be of any type. Would you like me to change that?